crates/stages/src/stages/headers.rs

```diff
@@ -36,11 +36,9 @@ const HEADERS: StageId = StageId("Headers");
 /// - [`HeaderNumbers`][reth_interfaces::db::tables::HeaderNumbers]
 /// - [`Headers`][reth_interfaces::db::tables::Headers]
 /// - [`CanonicalHeaders`][reth_interfaces::db::tables::CanonicalHeaders]
-/// - [`HeaderTD`][reth_interfaces::db::tables::HeaderTD]
 ///
-/// NOTE: This stage commits the header changes to the database (everything except the changes to
-/// [`HeaderTD`][reth_interfaces::db::tables::HeaderTD] table). The stage does not return the
-/// control flow to the pipeline in order to preserve the context of the chain tip.
```

Comment:
this comment has been obsolete after https://github.com/paradigmxyz/reth/pull/609